geomstats/geometry/riemannian_metric.py

```diff
@@ -453,6 +453,25 @@ def normal_basis(self, basis, base_point=None):
 
         return gs.einsum("i, ikl->ikl", 1.0 / gs.sqrt(norms), basis)
 
+    def covariant_riemannian_tensor(self, base_point):
+        r"""Compute purely covariant version of Riemannian tensor at base_point.
```

Comment:
Yes. I just checked because the tests were wrong but the convention used in the literature (or at least on wikipedia) is VERY weird (`R_{ijkl} = <R(x_j, x_k)x_i, x_l>`) so I think the previous successful tests were just a fluke. I will make a commit as soon as possible with your feedback!

Reply:
You must beware that the sign convention taken in Wikipedia for the Riemann curvature tensor (https://en.wikipedia.org/wiki/Riemann_curvature_tensor) is not the same as the one taken in geomstats (see [this docstring](https://github.com/geomstats/geomstats/blob/d1aa07d97cd5aa6b18373eb91d64969700c32463/geomstats/geometry/connection.py#L443)) and this has effects on the rest of the definitions. So all definitions should be taken from the *same reference book*, and not Wikipedia.

Reply:
Looking at all the formulae, I'm not completely sure that they are all compatible, i.e. take the same conventions. The only way to avoid mistakes in my view, is to pick a reference, and to take all formulas from this reference. This reference should be added to all docstrings.